platform/telemetry/src/TelemetryCapability.js

```diff
@@ -212,8 +212,10 @@ define(
             var telemetryAPI = this.openmct.telemetry;
 
             var metadata = telemetryAPI.getMetadata(domainObject);
-            var defaultDomain = (metadata.valuesForHints(['domain'])[0] || {}).key;
-            var defaultRange = (metadata.valuesForHints(['range'])[0] || {}).key;
+            var domainMetadata = metadata.valuesForHints(['x'])[0] || {};
+            var rangeMetadata = metadata.valuesForHints(['y'])[0] || {};
+            var defaultDomain = domainMetadata.source || domainMetadata.key;
```

Comment:
You can remove the ` || .key` for these two lines, the metadata manager ensures that valueMetadata always has a `source`:

https://github.com/nasa/openmct/blob/master/src/api/telemetry/TelemetryMetadataManager.js#L77-L85